python/dataproc_templates/pubsublite/pubsublite_to_gcs.py

```diff
@@ -113,16 +121,20 @@ def run(self, spark: SparkSession, args: Dict[str, Any]) -> None:
             .format(constants.FORMAT_PUBSUBLITE) \
             .option(f"{constants.FORMAT_PUBSUBLITE}.subscription", input_subscription_url) \
             .load())
+        
+        input_data = input_data.withColumn("data", input_data.data.cast(StringType()))
```

Comment:
1. casting data as StringType was done so that data in any of the file formats (csv included) aren't represented as byte strings
2. casting attributes as json was needed to write as a csv file, without which jobs were failing

Reply:
Nice, thanks.